### ISSUE TITLE
Including Next steps in README.md

### ISSUE THREAD

**Issue by Arshita-CHD** [conda-forge/conda-forge.github.io#1733]
### Where should the content be added?

https://github.com/conda-forge/conda-forge.github.io/blob/main/README.md

### What should be added?

After running `make html` command it is not mentioned what the next step is. 

For beginners, they might get confused on how to set up documentation web page locally so  I think we should steps like how to locally host (XAMPP for eg) and view pages in browser.

**Comment by Ginohmk** [comment]
@kathatherine Should I look into this issue ?, although I feel it has been fixed on Improved documentation to [setup project locally.](https://github.com/conda-forge/conda-forge.github.io/pull/1721) PR

**Comment by kathatherine** [comment]
@Ginohmk Yeah, it has. The README changes just haven't been merged in yet. Something we're going to do soon! So I think this ticket will be fixed then, but good to keep in mind.

**Comment by Ginohmk** [comment]
Alright 👍 @kathatherine  thanks for your reply
